### PR TITLE
railjson: update deadSection properties (railjson 3.3.1)

### LINKED ISSUE #4195: railjson: modify the DeadSection data model
The trackRanges of a DeadSection can not be common TrackRanges but ApplicableDirectionsTrackRanges.

### PR DESCRIPTION
**Pull request by clarani**
closes #4195

### CODE REVIEW

**Review by Castavo — APPROVED** [review]
Real nice ! I don't know if you really need to update the patch in `core` too tho, since it's just the patch... this PR wouldn't need a review from core maintainers then

**Review by younesschrifi — APPROVED** [review]

**Review by flomonster — APPROVED** [review]

**Review by eckter — APPROVED** [review]
Not an actual review, it's just for the "core maintainer" requirement (no core diff)

**Pull request merged**
